Wikipedia/Code/ReadingThemesControlsPresenter.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  ReadingThemesControlsPresenter.swift
+//  Wikipedia
+//
+//  Created by Toni Sevener on 2/27/19.
+//  Copyright © 2019 Wikimedia Foundation. All rights reserved.
+//
+
+import Foundation
+
+protocol ReadingThemesControlsPresenterProtocol: WMFReadingThemesControlsViewControllerDelegate, UIPopoverPresentationControllerDelegate {
```

Comment:
if you want to have a shorter name for the protocol, you could use `ReadingThemesControlsPresenting`, after https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingBasics.html

> Most protocols group related methods that aren’t associated with any class in particular. This type of protocol should be named so that the protocol won’t be confused with a class. A common convention is to use a gerund (“...ing”) form

Reply:
@nambatee oh yeah that's better, I'll do that. thx!